content/logs/index.md

```diff
@@ -130,15 +133,59 @@ logs:
 
 {{< /highlight >}}
 
+### Filter logs
+
+All logs are not equals and you might want to send only a specific subset of logs to Datadog.
+To achieve this use the `log_processing_rules` parameter in your configuration file with the **exclude_at_match** `type`
+
+If the pattern is contained in the message the log is filtered, and not sent to Datadog.
+
+Example: Filter logs with datadog users
+
+```
+logs:
+ - type: file
+   path: /my/test/file.log
+   service: cardpayment
+   source: java
+   log_processing_rules:
+    - type: exclude_at_match
+      name: exclude_datadoghq_users
+      # Regexp can be anything
+      pattern: User=\w+@datadoghq.com
+```
+
+### Search and replace content in your logs
+
+Logs can contain sensitive information that shouldn't leave your infrastructure without being redacted.
+To achieve this use the `log_processing_rules` parameter in your configuration file with the **mask_sequences** `type`
+
+It replace all matched groups with `replace_placeholder` parameter value.
```

Comment:
"This will replace" or "It will replace" or "This replaces"

Reply:
👍